couch/tests/test_couchv2.py

```diff
@@ -100,14 +101,14 @@ def test_check(aggregator, check, gauges, instance):
     for node in [common.NODE2, common.NODE3]:
         expected_tags = ["instance:{}".format(node["name"])]
         # One for the server stats, the version is already loaded
-        aggregator.assert_service_check(CouchDb.SERVICE_CHECK_NAME, status=CouchDb.OK, tags=expected_tags, count=1)
+        aggregator.assert_service_check(CouchDb.SERVICE_CHECK_NAME, status=CouchDb.OK, tags=expected_tags, count=2)
```

Comment:
Why the count changed ?

Reply:
Since we instantiate the class more than once, this always happens since no opportunity to cache:

https://github.com/DataDog/integrations-core/blob/36d441ed40163d2e7167dc7856699cc5d7314f57/couch/datadog_checks/couch/couch.py#L65